api/server-server/invites-v2.yaml

```
@@ -0,0 +1,234 @@
+# Copyright 2018-2019 New Vector Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+swagger: '2.0'
+info:
+  title: "Matrix Federation Invite User To Room API"
+  version: "1.0.0"
+host: localhost:8448
+schemes:
+  - https
+basePath: /_matrix/federation/v2
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  $ref: definitions/security.yaml
+paths:
+  "/invite/{roomId}/{eventId}":
+    put:
+      summary: Invites a remote user to a room
+      description: |-
+        .. Note::
+           This API is nearly identical to the v1 API with the exception of the request
+           body being different, and the response format fixed.
+
+        Invites a remote user to a room. Once the event has been  signed by both the inviting
+        homeserver and the invited homeserver, it can be sent to all of the servers in the
+        room by the inviting homeserver.
+
+        This endpoint is preferred over the v1 API as it is more useful for servers. Senders
+        which receive a 400 or 404 response to this endpoint may wish to retry using the v1
+        API as the server may be older, if the room version supports doing so.
```

```suggestion
        API as the server may be older, if the room version is "1" or "2".
```

I feel like we may as well be clear about which room versions, as we name then in other circumstances.